src/@orb.yml

```
@@ -0,0 +1,76 @@
+# CircleCI config for Auto Mobile Security Testing
+# See https://docs.nowsecure.com/auto/integration-services/jenkins-integration/
+# https://github.com/nowsecure/auto-circleci-plugin
+version: 2.1
+description: NowSecure AUTO provides fully automated, mobile appsec testing coverage
+display:
+  source_url: https://github.com/nowsecure/auto-circleci-plugin
+  home_url: https://www.nowsecure.com/
+commands:
+  mobile_security_test:
+    description: Executes security test using NowSecure AUTO.
+    parameters:
+      url:
+        description: url for nowsecure auto API
+        type: string
+        default: https://lab-api.nowsecure.com
+
+      token:
+        description: |
+          API token, visit
+          https://docs.nowsecure.com/auto/integration-services/jenkins-integration
+          to generate token
+        type: string
+        default: $AUTO_TOKEN
+
+      group:
+        description: Specify group if you belong to multiple groups
```

This is the group id not the group name. Right?
```suggestion
      group:
        description: Specify `group_id` if you belong to multiple groups
```

perhaps even a more drastic rename?

```suggestion
      group_id:
        description: Specify group uuid if you belong to multiple groups. You can obtain the group id by hovering over the group name in the [Account & Management > Groups](https://lab.nowsecure.com/account/management/groups)
```

---

Nice catch

---

I think changing the java source would better reflect this.